src/libs/actions/Report.js

```diff
@@ -412,6 +412,17 @@ function openReport(reportID, participantLoginList = [], newReportObject = {}, p
             value: {[optimisticCreatedAction.reportActionID]: {pendingAction: null}},
         });
 
+        // Add optimistic personal details for new participants
+        const optimisticPersonalDetails = {};
+        _.map(participantLoginList, (login, index) => {
+            optimisticPersonalDetails[newReportObject.participantAccountIDs[index]] = {login, displayName: login};
```

Comment:
That's vali,d but we can't do by `findBy`, since here we have two lists, `participantLoginList = ['test@email.com', 'test+1@email.com']` and `newReportObject.participantAccountIDs = [123123123, 345345345]`. I think the best solution would be to pass an object keyed by account ID but for that it would be necessary to change the function params

Reply:
@Beamanator forgot to tag, but what do you think about changing the function to receive an object instead of an array of logins (comment above)

Reply:
Sorry I didn't see your response 🙃 

Hmm actually looking at where we're calling `openReport`, I see that we basically create the optimistic `participantAccountIDs` for the new report in the same order we create the list of logins - so I don't really see how they could be mis-matched? Do you? I could have been over-thinking :D


One more thing: Should we add an avatar & accountID here?

Reply:
That was my logic, but this is taking into consideration that all `openReport` now and in the future will take that into consideration. For now I think it's fine, from what I see, all calls build `userLogins` using `getLoginsByAccountIDs` with `participantAccountIDs` or vice versa, so they should maintain the order.

> Should we add an avatar & accountID here? 
Perhaps it's better 😄 I'll add it